_data/projects/wattpad-scraper.yml

```diff
@@ -0,0 +1,12 @@
+name: Your Project Name Here
+desc: Easy to use wattpad scraper
+site: https://github.com/shhossain/wattpad-scraper
+tags:
+  - python
+  - package
+  - scraper
+  - wattpad
+  - test
+upforgrabs:
+  name: help wanted
+  link: https://github.com/shhossain/wattpad-scraper/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@shhossain Please change this to https://github.com/shhossain/wattpad-scraper/labels/help%20wanted

Reply:
link fixed